website/events/admin.py

```diff
@@ -127,6 +106,41 @@ class EventAdmin(DoNextTranslatedModelAdmin):
     prepopulated_fields = {"map_location": ("location",)}
     filter_horizontal = ("documents",)
 
+    fieldsets = (
+        (_("General"), {"fields": ("title", "published", "organiser",)}),
```

Comment:
I believe you're missing translations, but if you run those you will also have to make these changes probably: https://github.com/svthalia/concrexit/pull/1062/files#diff-830b743ee90a23a252b33004a915a561